### ISSUE TITLE
Add TestSuite Support for Rust

### ISSUE THREAD

**Issue by ALRubinger** [TBD54566975/developer.tbd.website#1193]
Add language and runtime support for Rust.

Some initial guides to get going:

* https://www.rust-lang.org/learn/get-started
* https://github.com/actions-rs
* https://github.com/actions-rs/example/blob/master/.github/workflows/quickstart.yml
* https://github.com/TBD54566975/web5-rs/blob/main/.github/workflows/ci.yml

Projects to enable:

* https://github.com/TBD54566975/tbdex-rs
* https://github.com/TBD54566975/web5-rs

**Comment by ALRubinger** [comment]
Will handle GitHub Actions and binding the test lifecycle later, when Rust SDKs are available, in #1195

**Comment by ALRubinger** [comment]
```
$> pnpm test:rust

> @ test:rust /Users/alr/git/TBD54566975/developer.tbd.website
> cd ./site/testsuites/testsuite-rust && cargo test

   Compiling testsuite-rust v0.1.0 (/Users/alr/git/TBD54566975/developer.tbd.website/site/testsuites/testsuite-rust)
    Finished test [unoptimized + debuginfo] target(s) in 0.85s
     Running unittests src/lib.rs (target/debug/deps/testsuite_rust-145f56dafcc88ff3)

running 1 test
test tests::it_works ... ok

test result: ok. 1 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.00s

   Doc-tests testsuite-rust

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.00s
```
